### PR TITLE
GCP - Expandr-6261

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-6261

## Description
Updated the to naming standard of the firewall rules to be in the format `remediation-<block|allow>-<vpc name>-port-<port nubmer>-<tcp|udp>`, where the VPC name is truncated to 30 characters to meet the overall 63 character limit for firewall rule names.  This ensures that unique firewall names are made in each VPC that a firewall rule could be created.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , if you can please review

File: Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_8.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### GCP - Firewall Remediation
+
+Updated the to naming standard of the firewall rules to be in the format `remediation-<block|allow>-<vpc name>-port-<port nubmer>-<tcp|udp>`, where the VPC name is truncated to 30 characters to meet the overall 63 character limit for firewall rule names.  This ensures that unique firewall names are made in each VPC that a firewall rule could be created.
```
Comment (ShirleyDenkberg) [inline Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_8.md:6]:
```suggestion
Updated the naming standard of the firewall rules to be in the format `remediation-<block|allow>-<vpc name>-port-<port nubmer>-<tcp|udp>`, where the VPC name is truncated to 30 characters to meet the overall 63 character limit for firewall rule names.  This ensures that unique firewall names are made in each VPC that a firewall rule could be created.
```

**Comment by ShirleyDenkberg** [comment]
@melamedbn @sapirshuker Doc review completed.

**Comment by sapirshuker** [comment]
Hi @johnnywilkes,
Thank you for your contribution. 
Your PR review will be slightly delayed technical problems, I apologize in advance. 
I'm available for any of your questions here or on slack.
Thanks again.

**Comment by melamedbn** [comment]
Hi @johnnywilkes,
Everything looks great. Can you please do a short demo (recording) of the playbook execution?

Thanks,
Ben

**Comment by johnnywilkes** [comment]
@melamedbn , i sent you a link from gdrive. Also, here is the Xpanse documentation I referenced in the video: https://docs-cortex.paloaltonetworks.com/r/Cortex-XPANSE/Cortex-Xpanse-Expander-User-Guide/Automated-Remediation-Capabilities-Matrix
image

**Review by melamedbn — APPROVED** [review]

**Comment by sapirshuker** [comment]
Build process failed it is a known issue: CIAC-6419

**Review by sapirshuker — APPROVED** [review]

**Pull request merged**